java/README.md

```diff
@@ -90,20 +58,130 @@ Other useful gradle developer commands:
 * `./gradlew :examples:run` to run client examples (make sure you have a running redis on port `6379`)
 * `./gradlew :benchmarks:run` to run performance benchmarks
 
-## Basic Examples
 
-### Standalone Redis:
+### Setting up the Driver
```

Comment:
I am finding this section confusing. I think that anyone wanting to contribute to this project should be referring to the `DEVELOPER.md` file for information. This section therefore seems to be addressed to _users_ of the client. 

When I follow the link in the first paragraph, that leads me to a Maven package repository. I am uncertain what this package is about and what depends on it. Earlier in this document, it says "the Java client must be built from source" at the moment. So, is the Maven package being explained here a dependency needed to _build_ the client or only to _use_ the client? What are the Maven coordinates of the package created by building the client?

I could possibly work this out by examining other files, but I believe this file should be relatively self-explanatory or at least refer people to where they can find more information. That suggests it might be beneficial to clarify what's meant here.